src/app/components/url-box/url-box.component.html

```diff
@@ -104,9 +104,12 @@
     [ngClass]="{ 'url-box__button--disabled': isSubscribed || isLoading }"
     track-id="set_request"
     *ngIf="queryOperations.length < 2"
+    style="line-height: normal;"
   >
     <span *ngIf="isLoading">{{ 'LOADING_INDICATOR_TEXT' | translate }}</span>
     <span *ngIf="!isLoading">{{ 'SEND_REQUEST_BUTTON' | translate }}</span>
+    <span class="button-tip-label" *ngIf="!isLoading && isMac">[Cmd + Enter]</span>
+    <span class="button-tip-label" *ngIf="!isLoading && !isMac">[Ctrl + Enter]</span>
```

Comment:
If possible, can this feed from:
https://github.com/imolorhe/altair/blob/staging/src/app/services/keybinder/keybinder.service.ts#L72

I'd much rather have one source of truth, rather than several shortcut details in different parts of the app. Makes it easier to manage and/or change.

Reply:
Done, but since we are using the same constant as the keybind, the mac shortcut became a little bit bigger:
![image](https://user-images.githubusercontent.com/8941938/66941012-5cfd6c80-f046-11e9-9615-89dda2d6e3cd.png)

Reply:
... and trimmed. What do you think?

Reply:
I agree it is bigger, but what other choice is there?